doc/PROTOCOL-WEB.md

```diff
@@ -39,6 +39,7 @@ Content-Type
   * e.g. application/grpc-web+[proto, json, thrift]
 2. application/grpc-web-text
   * text-encoded streams of “application/grpc-web”
+  * e.g. application/grpc-web+[proto, thrift]
```

Comment:
did you mean `application/grpc-web-text+proto` here, otherwise it's not different from the line above?